NEWS.md

```diff
@@ -105,6 +105,8 @@ unit = "s")
 
 4. `fifelse` and `fcase` notify users that S4 objects (except `nanotime`) are not supported [#4135](https://github.com/Rdatatable/data.table/issues/4135). Thanks to @torema-ed for bringing it to our attention and Morgan Jacob for the PR.
 
+5. Added documentation for the `stringsAsFactors` argument to address [#4706](https://github.com/Rdatatable/data.table/issues/4706). Thanks to @MichaelChirico for reporting and @markderry for the documentation.
```

Comment:
right now without looking at the issue it seems like stringsAsFactors is not documented at all. better to mention the issue a bit more specifically so users can just glance through the NEWS without clicking throigh